desktop-widgets/mapwidget.cpp

```diff
@@ -6,11 +6,14 @@
 #include "mapwidget.h"

```

Comment:
aww that's bad - poor GitHub UI. It's bc8f5c7401c1d417f4668bd42f01a50169d3d143 - but the comment shows up on every commit to that file... sorry about that.